prow/scripts/cluster-integration/kyma-gke-integration.sh

```diff
@@ -123,10 +125,26 @@ cleanup() {
     exit "${EXIT_STATUS}"
 }
 
-trap cleanup EXIT INT
+runTestLogCollector(){
+    if [ "${enableTestLogCollector}" = true ] ; then
+        if [[ "$BUILD_TYPE" == "master" ]]; then
+            shout "Install test-log-collector"
+            date
+            export PROW_JOB_TYPE="post-master-kyma-gke-integration"
+            ( 
+                # shellcheck disable=SC2030
+                export TEST_INFRA_SOURCES_DIR LOG_COLLECTOR_SLACK_TOKEN PROW_JOB_TYPE
```

Comment:
You don't need to export variables again in subshell. Once exported they will be copied to all child processes.